scripts/coverage_table.py

```
@@ -2,27 +2,57 @@
+
+Arguments:
+    output      The directory the coverage script wrote its reports into
+    threshold   The percentage the native tree is held to, 90 by default
 """
 
-import os
-import sys
-import json
+from json import loads
+from os.path import abspath, dirname, exists, join, relpath
+from sys import argv, exit
```

Ruff (0.16.2)

[error] Import `exit` is shadowing a Python builtin

(A004)